windows repo build errors (#9197)
carbon-design-system/carbon: .github/CONTRIBUTING.md

## Start contributing

## Setting up the repository:
## Windows guide:

> This repository runs best in a Linux envrionment, to avoid running into any errors as a windows user

Reviewer: ```suggestion
This repository runs best in a Linux environment. To avoid running into any errors as a Windows user
```

you will need to run the repo in a Linux environment using the Linux Subsystem for Windows.

System requirements:
https://docs.microsoft.com/en-us/windows/wsl/install-win10

prerequisites:
- aforementioned prerequisites
- Windows Subsystem for Linux

> - linux operating system of your choice.

Reviewer: ```suggestion
- Linux operating system distribution of your choice.
```

Author: The Microsoft guide is not user friendly. Would you just link to the git docs to get a user started working in the repo?

Reviewer: I wasn't suggesting to replace all of instructions with a link to the official WSL installation instructions, but I think we can replace the WSL installation portion with the link. I think it's reasonable to link the git docs to a user who is getting started with installing git. does that make more sense?

Author: Yeah if you think that would work better. Ideally the additions to the guide would be consistent with the current. Maybe including the link along with instructions would work better?

> ## 1: Open a powershell session and enter:

Reviewer: ```suggestion
## 1: Open a PowerShell session and enter:
```

Author: Sorry about the grammar, I whipped this up in a rush.

```
dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
```
## 2: Once the command has finished executing run:
```
dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
```

> ## 3: Install the Linux kernal update package

Reviewer: ```suggestion
## 3: Install the Linux kernel update package
```

https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi

> ## 4: Install the Ubuntu 18.04 from the microsoft store:

Reviewer: ```suggestion
## 4: Install Ubuntu 18.04 from the Microsoft Store:
```

https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q?activetab=pivot:overviewtab

> ### 4.5 Once installed, launch Ubuntu & setup your user account (follow the prompts)

Reviewer: ```suggestion
### 4.5 Once installed, launch Ubuntu & set up your user account (follow the prompts)
```

## 5: Run the commands (using Ubuntu) as follows.
1:
```
sudo apt-get update
```
2:
```
sudo apt-get upgrade
```
3: 

> Install linux build tools:

Reviewer: ```suggestion
Install Linux build tools:
```

```
sudo apt-get install build-essential
```
4:
```
sudo apt-get install autoconf automake gdb git libffi-dev zlib1g-dev libssl-dev
```
5:
Install Node.js 14
```
sudo curl -fsSL https://deb.nodesource.com/setup_14.x | sudo -E bash -

```
6:
```
sudo apt-get install -y nodejs
```
7: 
install Node version manager.
```
sudo curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
```
8:

> Install yarn

Reviewer: ```suggestion
Install Yarn
```
